src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -177,7 +177,7 @@ public function getDuration()
 
         for ($i = 0; $i < $left; ++$i) {
             $index = $stopped + $i;
-            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow());
+            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow(), true);
```

Comment:
@keradus See [all `func_get_arg` instances](https://github.com/symfony/symfony/search?utf8=%E2%9C%93&q=func_get_arg&type=) in Symfony. This practice is actually pretty novel in terms of providing new APIs without causing BC breaks.